src/constants/labels.json

```diff
@@ -0,0 +1,73 @@
+{
+  "overview": {
+    "welcome": "Willkommen bei Lunes!\nLerne Vokabeln für deinen Beruf.",
+    "unit": "Lektion",
+    "units": "Lektionen",
+    "word": "Wort",
+    "words": "Wörter",
+    "exercises": "Übungen"
+  },
+  "header": {
+    "overview": "Übersicht",
+    "overviewExercises": "Übersicht der Übungen",
+    "cancelExercise": "Übung beenden"
+  },
+  "exercises": {
+    "overview": {
+      "title": "Vokabelübersicht",
+      "description": "Alle Wörte anschauen"
+    },
+    "wordSingleChoice": {
+      "title": "Wort auswählen",
+      "description": "Wähle das richtige Wort"
+    },
+    "articleSingleChoice": {
+      "title": "Artikel üben",
+      "description": "Wähle den richtigen Artikel"
+    },
+    "write": {
+      "title": "Wort schreiben",
+      "description": "Wort mit Artikel eingeben",
+      "placeholder": {
```

Comment:
I'd not use these technical separations like `placeholder` and `button`. If you use another UI element for the same purpose it does not make sense anymore, so I think these constants should be completely independent of the concrete implementation. I guess I'd just omit the additional separation and put it one level up (directly in `write` in this example).

Reply:
done.